_layouts/digital-authorization-package-pilot.html

```diff
@@ -112,7 +112,7 @@ <h3 id="how-to-participate">How to Participate </h3>
       
 			<p>The pilot is open source, so there is no application or selection process for the pilot. To join, a CSP, GRC tool provider, or agency must <a href="https://app.smartsheetgov.com/b/form/6cf60ebf7d6844dc9c4d167062165354" target="_blank" rel="noopener noreferrer">sign up for the mailing list</a> and must have a GitHub account.</p>
 
-			<p>The FedRAMP automation team will host weekly OSCAL implementer meetings and recurring “pilot office hours”. To attend the weekly OSCAL implementer meetings, <a href="https://app.smartsheetgov.com/b/form/6cf60ebf7d6844dc9c4d167062165354" target="_blank" rel="noopener noreferrer">subscribe</a> to the distribution list. The pilot office hour times enable direct one-on-one discussions between an individual pilot partner and the FedRAMP team to support troubleshooting, debugging, or sharing detailed feedback. You can <a href="https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ27Rhb5mfcFTjMm7ITFc2xRbKnKdjjRS3ItLkehbzx9LGikIv11E2gjgh9bALmO9Pb4LOfaE5HA" target="_blank" rel="noopener noreferrer">sign up</a> for a 30-minute office hour time-block. These blocks are made available on a first-come, first served basis.</p>     
+			<p>The FedRAMP automation team will host biweekly OSCAL implementer meetings and recurring “pilot office hours”. For details on how to attend the biweekly OSCAL implementer meetings, please see our mailing list sign-up instructions <a href="https://automate.fedramp.gov/contact/#mailing-list" target="_blank" rel="noopener noreferrer">here</a>. The pilot office hour times enable direct one-on-one discussions between an individual pilot partner and the FedRAMP team to support troubleshooting, debugging, or sharing detailed feedback. You can <a href="https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ27Rhb5mfcFTjMm7ITFc2xRbKnKdjjRS3ItLkehbzx9LGikIv11E2gjgh9bALmO9Pb4LOfaE5HA" target="_blank" rel="noopener noreferrer">sign up</a> for a 30-minute office hour time-block. These blocks are made available on a first-come, first served basis.</p>     
```

Comment:
Minor capitalization issues but worth fixing now.

```suggestion
			<p>The FedRAMP automation team will host biweekly OSCAL Implementers meetings and recurring “pilot office hours”. For details on how to attend the biweekly OSCAL Implementers meetings, please see our mailing list sign-up instructions <a href="https://automate.fedramp.gov/contact/#mailing-list" target="_blank" rel="noopener noreferrer">here</a>. The pilot office hour times enable direct one-on-one discussions between an individual pilot partner and the FedRAMP Automation Team to support troubleshooting, debugging, or sharing detailed feedback. You can <a href="https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ27Rhb5mfcFTjMm7ITFc2xRbKnKdjjRS3ItLkehbzx9LGikIv11E2gjgh9bALmO9Pb4LOfaE5HA" target="_blank" rel="noopener noreferrer">sign up</a> for a 30-minute office hour time-block. These blocks are made available on a first-come, first served basis.</p>     
```